test/e2e/kubectl.go

```diff
@@ -61,6 +61,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/generated"
 	"k8s.io/kubernetes/test/e2e/scheduling"
+	. "k8s.io/kubernetes/test/e2e/workload"
```

Comment:
At the very lease, don't import with . so we can know where is the package used.